src/System.Net.Primitives/tests/FunctionalTests/CookieTest.cs

```diff
@@ -239,6 +239,13 @@ public static void Value_GetSet_Success()
             c.Value = null;
             Assert.Equal(string.Empty, c.Value);
         }
+        
+        [Fact]
```

Comment:
>Please add a comment specifying that this is not expected (and will fail) in .Net Desktop.

This is especially important because we will soon be activating these GitHub tests to run against net46 targets in addition to the current CoreFx targets.  And when we do that, the test will fail.  We'll then need to add some special code to differentiate the expected results.

So, having a comment here will be very helpful to remind us about this.